src/adhocracy_core/adhocracy_core/evolution/__init__.py

```diff
@@ -506,6 +508,30 @@ def update_asset_download_children(root):  # pragma: no cover
             logger.warn('Asset {} has no downloads to migrate.'.format(asset))
 
 
+@log_migration
+def migrate_all_image_size_downloads_to_jpeg(root):  # pragma: no cover
+    """Migrate all image size downloads to jpeg images.
+
+    This happens automatically if we recreate them.
+    """
+    from adhocracy_core.sheets.asset import IAssetMetadata
+    from adhocracy_core.sheets.image import IImageMetadata
+    from adhocracy_core.resources.image import add_image_size_downloads
+    from adhocracy_core.resources.image import IImageDownload
+    registry = get_current_registry(root)
+    catalogs = find_service(root, 'catalogs')
+    assets = _search_for_interfaces(catalogs, IAssetMetadata)
+    images = [x for x in assets if IImageMetadata.providedBy(x)]
```

Comment:
> why not search for IImageMetadata in the _search_for_interfaces function?

We missed reindexing the interfaces index, wo the search for IImageMetadata resturns no result on the mercator platform.